docs/source/model_sharing.rst

```diff
@@ -98,8 +98,10 @@ with any other git repo.
     echo "hello" >> README.md
     git add . && git commit -m "Update from $USER"
 
-We are intentionally not wrapping git too much, so as to stay intuitive and easy-to-use.
+We are intentionally not wrapping git too much, so that you can pick the workflow that you already know.
```

Comment:
```suggestion
We are intentionally not wrapping git too much, so that you can go on with the workflow you're used to and the tools you already know.
```